readme.md

```diff
@@ -5,3 +5,12 @@ The main goal of this tool is to have some visual and geographical feedback on [
 :construction::warning: This is a work in progress, and deeply connected to the Cosmogony output format :construction::warning:
 
 ![example](cosmogony_explorer.gif)
+
+## Install
+
+* Get some cosmogony output file (as `cosmogony.json`)
+* Create a geojson out of it (`python3 cosmogony_to_geojson.py`)
+* Import the geojson in a postgis database (`ogr2ogr -f "PostgreSQL" PG:"dbname=my_database user=postgres" "cosmogony.geojson"`)
```

Comment:
I wish I had the time to make a direct cosmogony to postgis option :)

Reply:
that would be great 👍